torch/csrc/utils/python_arg_parser.h

```diff
@@ -246,6 +247,11 @@ inline c10::optional<at::Scalar> PythonArgs::scalarOptional(int i) {
   return scalar(i);
 }
 
+inline c10::optional<int64_t> PythonArgs::int64_tOptional(int i) {
```

Comment:
Aha, looks like I already add the optional<int> in #15234, so you should already be able to use it :)